src/mdViews/patientOverview/StatusComponents/RecommendationsWidget.tsx

```diff
@@ -1,16 +1,111 @@
-import { FC } from 'react';
-import { Flex, useColorModeValue } from '@chakra-ui/react';
+import React, { FC } from 'react';
+import { Badge, Button, Divider, Flex, Stack, Text } from '@chakra-ui/react';
+import { Link, useParams } from 'react-router-dom';
+import { ImmunizationRecommendation } from '../../../core/models';
+import {
+  useImmunizationRecommendations,
+  useMedicationInfo,
+  useMedications,
+  useTargetDiseases,
+} from '../../../hooks';
 
 export const RecommendationsWidget: FC = ({}) => {
+  const params = useParams();
+  const patientId = params['patientId'];
+
+  const { data: targetDiseases, idToTargetDisease } = useTargetDiseases({});
+  const { immunizationRecommendations } = useImmunizationRecommendations({
+    patient: patientId,
+  });
+
+  const { data: medicationsData, idToMedication } = useMedications({});
+  const {
+    vaccinationSchemes,
+    idToVaccinationScheme,
+    vaccinationDoses,
+    idToVaccinationDose,
+  } = useMedicationInfo(
+    medicationsData?.ids.map((id: string) => idToMedication(id)!)
+  );
+
+  const standardVaccinationSchemes = vaccinationSchemes?.byType['standard'];
+
   return (
     <Flex
-      bg={useColorModeValue('white', 'gray.900')}
+      bg={'white'}
       boxShadow='0 4px 12px 0 rgba(0, 0, 0, 0.15)'
       borderRadius={'15px'}
       flexDir='column'
-      justifyContent='space-between'
+      justifyContent='start'
       h={'100%'}
       w={'100%'}
-    ></Flex>
+      overflowY={'scroll'}
+    >
+      {immunizationRecommendations?.map(
+        (recommendation: ImmunizationRecommendation) => {
+          const diseases = targetDiseases?.byCode[
+            recommendation.targetDisease.coding.code
+          ]?.ids.map((irId) => idToTargetDisease(irId)!);
+          return diseases?.map((disease) => {
+            const med =
+              recommendation &&
+              idToMedication(
+                medicationsData?.byCode[recommendation.vaccineCode.coding.code]
+                  ?.ids[0]
+              );
+            const vs =
+              med &&
+              idToVaccinationScheme(
+                standardVaccinationSchemes?.byMedication[med?.id]?.ids[0]
+              );
+            const allDoses =
+              vs &&
+              vaccinationDoses?.byVaccinationScheme[vs.id]?.ids.map(
+                idToVaccinationDose
+              );
+            const dose = idToVaccinationDose(recommendation?.vaccinationDoseId);
```

Comment:
```suggestion
            const dose = idToVaccinationDose(recommendation?.vaccinationDoseId);
            const vs = dose && idToVaccinationScheme(dose?.vaccinationSchemeId);
            const allDoses =
              vs &&
              vaccinationDoses?.byVaccinationScheme[vs.id]?.ids.map(
                idToVaccinationDose
              );
```
This is incorrect, we need to retrieve the exact scheme the dose we recommend belongs to, which is not necessarily the standard scheme.